wrapanapi/lenovo.py

```diff
@@ -21,22 +25,44 @@ class LenovoSystem(WrapanapiAPIBase):
         'num_server': lambda self: len(self.list_server()),
     }
 
-    def __init__(self, hostname, username, password, **kwargs):
-        super(LenovoSystem, self).__init__(kwargs)
+    def __init__(self, hostname, username, password, protocol="https", port=443, **kwargs):
         self.hostname = hostname
+        self.port = port
         self.username = username
         self.password = password
+        self.protocol = protocol
+        self._content = None
 
     def __del__(self):
         """Disconnect from the API when the object is deleted"""
         # This isn't the best place for this, but this class doesn't know when it is no longer in
         # use, and we need to do some sort of disconnect based on the pyVmomi documentation.
-        raise NotImplementedError
+        self.port = self.port
```

Comment:
This is essential a no-op here - is there a disconnect method to end the session?